media-libs/oidn/oidn-2.3.0.ebuild

```diff
@@ -86,7 +88,7 @@ src_configure() {
 	if use hip; then
 		mycmakeargs+=(
 			-DROCM_PATH="${EPREFIX}/usr"
-			-DOIDN_DEVICE_HIP_COMPILER="$(get_llvm_prefix)/bin/clang++" # use HIPHOSTCOMPILER
+			-DOIDN_DEVICE_HIP_COMPILER="${ESYSROOT}/usr/bin/hipcc"
```

Comment:
I guess we still have no better way of getting the path to hipcc than this hard coded path at this point?

Reply:
Yes, I don't see a reason to migrate from `/usr/bin/hipcc` for now.

Reply:
Sounds good :+1: